doc/user_doc/vic_admin/upgrade_vch.md

```diff
@@ -42,10 +50,9 @@ For descriptions of the options that `vic-machine upgrade` includes in addition
 During the upgrade process, `vic-machine upgrade` performs the following operations:
 
 - Validates whether the configuration of the existing VCH is compatible the new version. If not, the upgrade fails. 
```

Comment:
Done in https://github.com/vmware/vic/pull/4144/commits/2c07b3d163be8e6e0c9aa3b9d76746eb9c8875bd